src/components/YouTube.js

```diff
@@ -0,0 +1,103 @@
+import React, {useState} from 'react'
+import styled from 'styled-components'
+
+const YouTubeStyled = styled.div`
+padding: 120px;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: flex-start;
+
+
+.SearchBar{
+    width: 100%;
+}
+
+.searchButton{
+    width: 100%;
+}
+
+.cardDisplay{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.videoCard{
```

Comment:
Here it's a good idea to create a folder called `Youtube`, and then create 3 files inside it:

1. **Youtube.component.jsx**: this is the place for the logic, the state, the html of the component
2. **Youtube.styles.jsx**: the content of this file will be the styles used in this component, in this case what is stored in the variable `YouTubeStyled`
3. **index.js**: this is just `export { default } from './Youtube.component';`, this way you can import the component by using `import Youtube from '../path/to/components/Youtube'`